js/node/CMakeLists.txt

```diff
@@ -86,61 +96,29 @@ else()
 endif()
 
 if (WIN32)
-  target_link_libraries(onnxruntime_binding PRIVATE onnxruntime.lib)
+  target_link_libraries(onnxruntime_binding PRIVATE onnxruntime)
 elseif (APPLE)
   target_link_libraries(onnxruntime_binding PRIVATE libonnxruntime.${ort_version}.dylib)
   set_target_properties(onnxruntime_binding PROPERTIES INSTALL_RPATH "@loader_path")
 else()
-  target_link_libraries(onnxruntime_binding PRIVATE libonnxruntime.so.${ort_version})
```

Comment:
In the old code, even though we specify the library with a full version number,  the version number is only used at linking time, not runtime.  At runtime onnxruntime_binding still depends on libonnxruntime.so.1 instead of libonnxruntime.so.${ort_version}. Because there is a field in libonnxruntime.so.${ort_version} says so.